instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/OpenTelemetryAutoConfiguration.java

```diff
@@ -76,7 +76,7 @@ public MapConverter mapConverter() {
       return new MapConverter();
     }
 
-    @Bean
+    @Bean(destroyMethod = "")
```

Comment:
maybe something like?
```suggestion
    @Bean(destroyMethod = "") // SDK components are shutdown from the OpenTelemetry instance
```